### PR TITLE
lkmex transfer events

### PR DESCRIPTION
**Pull request by psorinionut**

### CODE REVIEW

**Review by ovidiuolteanu — APPROVED** [review]

**Review by CostinCarabas — APPROVED** [review]

File: locked-asset/lkmex-transfer/src/events.rs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        locked_funds: LockedFunds<Self::Api>,
+    );
+
+    #[event("lock_funds_event")]
+    fn lock_funds_event(
+        &self,
+        #[indexed] sender: ManagedAddress,
+        #[indexed] receiver: ManagedAddress,
+        #[indexed] current_epoch: Epoch,
+        payments: PaymentsVec<Self::Api>,
```
Comment (claudiulataretu) [inline locked-asset/lkmex-transfer/src/events.rs:58]: use `LockedFunds` struct. Don't index `current_epoch`

File: locked-asset/lkmex-transfer/src/events.rs [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    ) {
+        self.lock_funds_event(sender, receiver, current_epoch, payments);
+    }
+
+    #[event("withdraw_event")]
+    fn withdraw_event(
+        &self,
+        #[indexed] sender: ManagedAddress,
+        #[indexed] receiver: ManagedAddress,
+        payments: PaymentsVec<Self::Api>,
```
Comment (claudiulataretu) [inline locked-asset/lkmex-transfer/src/events.rs:41]: use `LockedFunds` struct.

Comment (psorinionut) [reply locked-asset/lkmex-transfer/src/events.rs]: Done.

File: locked-asset/lkmex-transfer/src/events.rs (hunk 1 above)
Comment (psorinionut) [reply locked-asset/lkmex-transfer/src/events.rs]: Done.

**Review by claudiulataretu — APPROVED** [review]

**Pull request merged**